2024-11-14 01:49:56,549 INFO    StreamThr :46380 [internal.py:wandb_internal():85] W&B internal server running at pid: 46380, started at: 2024-11-14 01:49:56.548847
2024-11-14 01:49:56,550 DEBUG   HandlerThread:46380 [handler.py:handle_request():158] handle_request: status
2024-11-14 01:49:56,559 INFO    WriterThread:46380 [datastore.py:open_for_write():87] open: /Users/dan/Documents/src/COMP-STUDY/synced/wandb/run-20241114_014956-yy1pml1w/run-yy1pml1w.wandb
2024-11-14 01:49:56,560 DEBUG   SenderThread:46380 [sender.py:send():389] send: header
2024-11-14 01:49:56,575 DEBUG   SenderThread:46380 [sender.py:send():389] send: run
2024-11-14 01:49:57,211 INFO    SenderThread:46380 [dir_watcher.py:__init__():211] watching files in: /Users/dan/Documents/src/COMP-STUDY/synced/wandb/run-20241114_014956-yy1pml1w/files
2024-11-14 01:49:57,211 INFO    SenderThread:46380 [sender.py:_start_run_threads():1157] run started: yy1pml1w with start time 1731545396.548534
2024-11-14 01:49:57,226 DEBUG   HandlerThread:46380 [handler.py:handle_request():158] handle_request: run_start
2024-11-14 01:49:57,233 DEBUG   HandlerThread:46380 [system_info.py:__init__():26] System info init
2024-11-14 01:49:57,233 DEBUG   HandlerThread:46380 [system_info.py:__init__():41] System info init done
2024-11-14 01:49:57,233 INFO    HandlerThread:46380 [system_monitor.py:start():194] Starting system monitor
2024-11-14 01:49:57,233 INFO    SystemMonitor:46380 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-11-14 01:49:57,233 INFO    HandlerThread:46380 [system_monitor.py:probe():214] Collecting system info
2024-11-14 01:49:57,234 INFO    SystemMonitor:46380 [interfaces.py:start():188] Started cpu monitoring
2024-11-14 01:49:57,236 INFO    SystemMonitor:46380 [interfaces.py:start():188] Started disk monitoring
2024-11-14 01:49:57,241 INFO    SystemMonitor:46380 [interfaces.py:start():188] Started gpuapple monitoring
2024-11-14 01:49:57,242 INFO    SystemMonitor:46380 [interfaces.py:start():188] Started memory monitoring
2024-11-14 01:49:57,243 INFO    SystemMonitor:46380 [interfaces.py:start():188] Started network monitoring
2024-11-14 01:50:57,247 DEBUG   SystemMonitor:46380 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-11-14 02:39:44,079 WARNING FileStreamThread:46380 [file_stream.py:request_with_retry():674] requests_with_retry encountered retryable exception: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response')). func: <bound method Session.post of <requests.sessions.Session object at 0x104649190>>, args: ('https://api.wandb.ai/files/ctufit/pelanekd-ALcompstudy-test/yy1pml1w/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2024-11-14 03:58:41,817 WARNING FileStreamThread:46380 [file_stream.py:request_with_retry():674] requests_with_retry encountered retryable exception: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response')). func: <bound method Session.post of <requests.sessions.Session object at 0x104649190>>, args: ('https://api.wandb.ai/files/ctufit/pelanekd-ALcompstudy-test/yy1pml1w/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2024-11-14 05:35:05,014 WARNING FileStreamThread:46380 [file_stream.py:request_with_retry():674] requests_with_retry encountered retryable exception: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response')). func: <bound method Session.post of <requests.sessions.Session object at 0x104649190>>, args: ('https://api.wandb.ai/files/ctufit/pelanekd-ALcompstudy-test/yy1pml1w/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2024-11-14 06:15:24,660 WARNING FileStreamThread:46380 [file_stream.py:request_with_retry():674] requests_with_retry encountered retryable exception: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response')). func: <bound method Session.post of <requests.sessions.Session object at 0x104649190>>, args: ('https://api.wandb.ai/files/ctufit/pelanekd-ALcompstudy-test/yy1pml1w/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2024-11-14 07:38:46,432 WARNING FileStreamThread:46380 [file_stream.py:request_with_retry():674] requests_with_retry encountered retryable exception: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response')). func: <bound method Session.post of <requests.sessions.Session object at 0x104649190>>, args: ('https://api.wandb.ai/files/ctufit/pelanekd-ALcompstudy-test/yy1pml1w/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2024-11-14 08:58:29,973 WARNING FileStreamThread:46380 [file_stream.py:request_with_retry():674] requests_with_retry encountered retryable exception: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response')). func: <bound method Session.post of <requests.sessions.Session object at 0x104649190>>, args: ('https://api.wandb.ai/files/ctufit/pelanekd-ALcompstudy-test/yy1pml1w/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2024-11-14 09:56:10,220 WARNING FileStreamThread:46380 [file_stream.py:request_with_retry():674] requests_with_retry encountered retryable exception: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response')). func: <bound method Session.post of <requests.sessions.Session object at 0x104649190>>, args: ('https://api.wandb.ai/files/ctufit/pelanekd-ALcompstudy-test/yy1pml1w/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2024-11-14 11:27:31,827 INFO    cpu       :46380 [interfaces.py:monitor():138] Process cpu has exited.
2024-11-14 11:27:31,828 DEBUG   SystemMonitor:46380 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-11-14 11:27:31,828 DEBUG   SystemMonitor:46380 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-11-14 11:27:32,255 WARNING FileStreamThread:46380 [file_stream.py:request_with_retry():674] requests_with_retry encountered retryable exception: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response')). func: <bound method Session.post of <requests.sessions.Session object at 0x104649190>>, args: ('https://api.wandb.ai/files/ctufit/pelanekd-ALcompstudy-test/yy1pml1w/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2024-11-14 11:27:35,097 WARNING StreamThr :46380 [internal.py:is_dead():413] Internal process exiting, parent pid 46354 disappeared
2024-11-14 11:27:35,097 ERROR   StreamThr :46380 [internal.py:wandb_internal():151] Internal process shutdown.
2024-11-14 11:27:35,108 INFO    SenderThread:46380 [sender.py:finish():1589] shutting down sender
2024-11-14 11:27:35,108 INFO    SenderThread:46380 [dir_watcher.py:finish():358] shutting down directory watcher
2024-11-14 11:27:36,101 INFO    WriterThread:46380 [datastore.py:close():296] close: /Users/dan/Documents/src/COMP-STUDY/synced/wandb/run-20241114_014956-yy1pml1w/run-yy1pml1w.wandb
2024-11-14 11:27:36,112 INFO    SenderThread:46380 [dir_watcher.py:finish():388] scan: /Users/dan/Documents/src/COMP-STUDY/synced/wandb/run-20241114_014956-yy1pml1w/files
2024-11-14 11:27:36,113 INFO    SenderThread:46380 [dir_watcher.py:finish():402] scan save: /Users/dan/Documents/src/COMP-STUDY/synced/wandb/run-20241114_014956-yy1pml1w/files/config.yaml config.yaml
2024-11-14 11:27:36,113 INFO    SenderThread:46380 [file_pusher.py:finish():169] shutting down file pusher
2024-11-14 11:27:36,114 INFO    SenderThread:46380 [file_pusher.py:join():175] waiting for file pusher
2024-11-14 11:27:37,857 INFO    wandb-upload_0:46380 [upload_job.py:push():130] Uploaded file /Users/dan/Documents/src/COMP-STUDY/synced/wandb/run-20241114_014956-yy1pml1w/files/config.yaml
2024-11-14 11:27:38,133 INFO    SenderThread:46380 [file_stream.py:finish():601] file stream finish called
2024-11-14 11:27:38,387 INFO    SenderThread:46380 [file_stream.py:finish():605] file stream finish is done
2024-11-14 11:49:50,833 ERROR   HandlerThread:46380 [gpu_apple.py:probe():176] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 164, in probe
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-11-14 11:49:50,881 DEBUG   HandlerThread:46380 [system_info.py:probe():152] Probing system
2024-11-14 11:49:50,853 ERROR   gpuapple  :46380 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-11-14 11:49:50,933 DEBUG   HandlerThread:46380 [system_info.py:_probe_git():137] Probing git
2024-11-14 11:49:52,443 DEBUG   HandlerThread:46380 [system_info.py:_probe_git():145] Probing git done
2024-11-14 11:49:52,443 DEBUG   HandlerThread:46380 [system_info.py:probe():200] Probing system done
2024-11-14 11:49:52,444 DEBUG   HandlerThread:46380 [system_monitor.py:probe():223] {'os': 'macOS-13.4.1-arm64-arm-64bit', 'python': '3.11.1', 'heartbeatAt': '2024-11-14T10:49:50.882498', 'startedAt': '2024-11-14T00:49:56.536360', 'docker': None, 'cuda': None, 'args': ('--config-name', 'lgbm-conf'), 'state': 'running', 'program': '/Users/dan/Documents/src/COMP-STUDY/synced/uncertainty_strategy.py', 'codePathLocal': 'uncertainty_strategy.py', 'codePath': 'uncertainty_strategy.py', 'git': {'remote': 'git@github.com:Siigull/AL-Comparative-Study.git', 'commit': '7e7a49d7642bb2c5d4879014e26f7ca35fa0c5ae'}, 'email': None, 'root': '/Users/dan/Documents/src/COMP-STUDY/synced', 'host': 'Daniels-MacBook-Air-2.local', 'username': 'dan', 'executable': '/Library/Frameworks/Python.framework/Versions/3.11/bin/python3', 'cpu_count': 8, 'cpu_count_logical': 8, 'cpu_freq': {'current': 3204, 'min': 600, 'max': 3204}, 'cpu_freq_per_core': [{'current': 3204, 'min': 600, 'max': 3204}], 'disk': {'/': {'total': 228.27386474609375, 'used': 11.198070526123047}}, 'gpuapple': {'type': 'arm', 'vendor': 'Apple'}, 'memory': {'total': 16.0}}
2024-11-14 11:49:52,444 INFO    HandlerThread:46380 [system_monitor.py:probe():224] Finished collecting system info
2024-11-14 11:49:52,444 INFO    HandlerThread:46380 [system_monitor.py:probe():227] Publishing system info
2024-11-14 11:49:52,445 INFO    HandlerThread:46380 [system_monitor.py:probe():229] Finished publishing system info
2024-11-14 11:49:52,450 INFO    HandlerThread:46380 [handler.py:finish():874] shutting down handler
2024-11-14 11:49:52,450 INFO    HandlerThread:46380 [system_monitor.py:finish():203] Stopping system monitor
2024-11-14 11:49:52,450 INFO    HandlerThread:46380 [interfaces.py:finish():200] Joined cpu monitor
2024-11-14 11:49:52,450 INFO    HandlerThread:46380 [interfaces.py:finish():200] Joined disk monitor
2024-11-14 11:49:52,450 INFO    HandlerThread:46380 [interfaces.py:finish():200] Joined gpuapple monitor
2024-11-14 11:49:52,450 INFO    HandlerThread:46380 [interfaces.py:finish():200] Joined memory monitor
2024-11-14 11:49:52,450 INFO    HandlerThread:46380 [interfaces.py:finish():200] Joined network monitor
